service/ocean/providers/gcp/cluster.go

```diff
@@ -1420,3 +1436,54 @@ func (o *LaunchSpecShieldedInstanceConfig) SetEnableSecureBoot(v *bool) *LaunchS
 }
 
 //endregion
+
+//region cluster clusterRoll
```

Comment:
interchange the order, keep parameters first and then clusterRoll

Reply:
Updated.